rllib/algorithms/impala/tf/impala_tf_learner.py

```diff
@@ -143,16 +80,19 @@ def compute_loss_per_module(
             discounts=discounts_time_major,
         )
 
-        batch_size = tf.cast(target_actions_logp_time_major.shape[-1], tf.float32)
+        # Sample size is T x B, where T is the trajectory length and B is the batch size
+        sample_size = tf.cast(
+            tf.math.reduce_prod(target_actions_logp_time_major.shape[:1]), tf.float32
+        )
```

Comment:
@avnishn @kouroshHakha I argue that calculating the mean value per TS is more meaningful here.

Reply:
As we chatted offline, let's keep the regression minimal (close to none) compared to the old stack. add a todo about what needs to be done here, for later.